credmark/cmf/types/series.py

```diff
@@ -67,6 +69,39 @@ def get(self, block_number=None, timestamp=None) -> Union[BlockSeriesRow[DTOCLS]
                                  [s for s in self.series if s.blockTimestamp <= timestamp]))
         return None
 
+    def to_list(self, fields: Optional[List[Callable]] = None) -> List[List]:
```

Comment:
This could maybe be pushed to a higher superclass than series. perhaps I'm wrong?

Reply:
`.to_list()` can be added to the higher level parent class, i.e. `IterableListGenericDTO()`. 

Whether it is useful depends on how we consume the data.
- For things like Portfolio, we can iterate through the list of assets.
- For things like Series, we need to use the entire data to calculate some properties, `.to_list()` and `.to_dataframe()` are useful.